src/hello.js

```diff
@@ -1729,7 +1729,7 @@ hello.api = function() {
 	// Extrapolate the QueryString
 	// Provide a clean path
 	// Move the querystring into the data
-	if (p.method === 'get') {
+	if (p.method === 'get' || p.method === 'post') {
```

Comment:
This will need tests

Reply:
Thanks for your feedback. I've updated the pull request. I had a hard time writing meaningful tests since most of it already seemed covered.